docs/src/prototypes/customToolbar/CustomToolbar.tsx

```diff
@@ -0,0 +1,341 @@
+//  - [x] Add kind: 'custom', non-focusable container that renders any control
+//  - [ ] Add all toolbar items (menu, etc)
+//  - [ ] Consider exporting IS_FOCUSABLE_ATTRIBUTE, it is also shown in our accessibility docs
+
+export interface CustomToolbarProps {
+  layout?: 'standard' | 'desktop-share' | 'powerpoint-presenter'
+
+  isRecording?: boolean
+
+  cameraActive?: boolean
+  onCameraChange?: (state: boolean) => void
+
+  micActive?: boolean
+  onMicChange?: (state: boolean) => void
+
+  screenShareActive?: boolean
+  onScreenShareChange?: (state: boolean) => void
+
+  sidebarSelected: false | 'chat' | 'participant-add'
+  onSidebarChange?: (state: false | 'chat' | 'participant-add') => void
+
+  chatHasDot?: boolean
+
+  pptSlide?: string
+  onPptPrevClick?: () => void
+  onPptNextClick?: () => void
+
+  onEndCallClick?: () => void
+}
+
+type CustomToolbarItem =
+  | ((ToolbarItemProps | ToolbarCustomItemProps) & {
+      as?: any
+      key: string
+      kind?: ToolbarItemShorthandKinds
+    })
+  | ((render: any) => any)
+type CustomToolbarLayout = (props: CustomToolbarProps) => CustomToolbarItem[]
+
+const FocusableFlexBox = ({ variables = undefined, ...props }) => {
+  const [fromKeyboard, setFromKeyboard] = React.useState(false)
+
+  // TODO: this is possible with investigation see index.ejs
+  // return (
+  //   <div
+  //     tabIndex={0}
+  //     data-is-focusable
+  //     {...props}
+  //   />
+  //
+  // )
+
+  // TODO: conceal in a focusable component or wrapper
+  return (
+    <Flex
+      data-is-focusable
+      {...props}
+      variables={{ focusable: true, isFromKeyboard: fromKeyboard, ...variables }}
+      onFocus={(...args) => {
+        setFromKeyboard(isFromKeyboard())
+        _.invoke(props, 'onFocus', args)
+      }}
+    />
+  )
+}
+
+const FocusableStatus = ({ variables = undefined, ...props }) => {
+  const [fromKeyboard, setFromKeyboard] = React.useState(false)
+
+  return (
+    <Status
+      data-is-focusable
+      {...props}
+      variables={{ focusable: true, isFromKeyboard: fromKeyboard, ...variables }}
+      onFocus={(...args) => {
+        setFromKeyboard(isFromKeyboard())
+        _.invoke(props, 'onFocus', args)
+      }}
+    />
+  )
+}
+
+const commonLayout: CustomToolbarLayout = props => [
+  ...((props.isRecording
+    ? [
+        // Status has focus
+        // {
+        //   key: 'recording1',
+        //   kind: 'custom',
+        //   fitted: 'horizontally',
+        //   content: <FocusableStatus state="error" title="Recording" styles={{ margin: '0 9px' }} />,
+        //   variables: { primary: true },
+        // },
+
+        // TODO: Winner. Meets visual parity with Teams and code is most sane.  Confirm acc with Juraj.
+        // Flexbox has focus
+        {
+          key: 'recording2',
+          kind: 'custom',
+          fitted: 'horizontally',
+          content: (
+            <FocusableFlexBox fill gap="gap.medium" vAlign="center">
```

Comment:
important thing here is that if the focus lands on the FocusableFlexBox div, the onClick handler should also be there (and we will need to add keyboard handlers for enter/space as well as aria role). Based on this:
- can we prevent the user from putting onClick to the 'custom' kind using types or prevent spreading it to the root?
- should we rename it to ClickableFlexBox?

I understand that in this particular use case they do not use onClick, but if this will become a generic thing, it should handle it correctly.

Side note - if this would have popup/menu/tooltip attached, the FocusableFlexBox should become the trigger for it.
But maybe we will just say that custom items cannot have popup/menu/tooltip and we will let consumers handle it - in that case the type should also be limited appropriately and not expose those props.